config/config.js

```diff
@@ -4,22 +4,26 @@ const dotenv = require("dotenv");
 dotenv.config();
 
 const envSchema = Joi.object({
-	NODE_ENV: Joi.string()
-		.allow(["development", "production", "test", "provision"])
-		.default("development"),
-	PORT: Joi.number().default(3000),
-	MONGODB_HOST: Joi.string().required(),
-	JWT_SECRET: Joi.string().required()
+  NODE_ENV: Joi.string()
+    .allow(["development", "production", "test", "provision"])
```

Comment:
What do you mean by provision?

Reply:
orvi vai added this... i know the first three terms.

Reply:
provision delete kore dao